Jenkinsfile

```diff
@@ -15,7 +15,7 @@ pipeline {
 
             steps {
                 script {
-                    TAG = "${BRANCH_NAME}"
+                    TAG=BRANCH_NAME.split('/').last()
```

Comment:
could this possibly cause a collision if the branch name after the slash has been used before?

Reply:
yeah it would, but also uses the "PR" branch that Github creates, which is meant to be unique.

I added that split just to avoid errors.

https://cloud.docker.com/u/adhocteam/repository/registry-1.docker.io/adhocteam/jenkins/tags will show the branch labels and the pr labels.

I figured there may be cases like some proposal demos were we want a "named" tag on there but wouldn't open a PR. for the PR case, we would use the PR tag because jenkins only runs the branch the first time. after that it only builds the PR